src/command_modules/azure-cli-monitor/setup.py

```diff
@@ -0,0 +1,55 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+from codecs import open
+from setuptools import setup
+
+VERSION = '0.1.1b2+dev'
+
+CLASSIFIERS = [
+    'Development Status :: 4 - Beta',
+    'Intended Audience :: Developers',
+    'Intended Audience :: System Administrators',
+    'Programming Language :: Python',
+    'Programming Language :: Python :: 2',
+    'Programming Language :: Python :: 2.7',
+    'Programming Language :: Python :: 3',
+    'Programming Language :: Python :: 3.4',
+    'Programming Language :: Python :: 3.5',
+    'Programming Language :: Python :: 3.6',
+    'License :: OSI Approved :: MIT License',
+]
+
+DEPENDENCIES = [
+    'azure-cli-core',
+    'azure-monitor==0.1.0'
+    # 'azure-mgmt-monitor==0.1.0'
```

Comment:
@derekbekoe I was thinking that we'll not merge the PR in as it has sdk checked-in. So I was planning to wait until azure-mgmt-monitor releases and then removing sdks and the merging PR. 

Let me know if there is way where we still allow merging PRs if sdks are checked-in? If so i'll definitely remove it.